inductiva/resources/machines_base.py

```diff
@@ -160,6 +173,16 @@ def _convert_auto_terminate_ts(
 
         return None
 
+    def _disk_config_to_dict(self):
+        if self.disk_config is None or not self.disk_config.is_resizable:
+            return None
+
+        return {
+            "free_space_threshold_gb": self.disk_config.resize_trigger_gb,
+            "size_increment_gb": self.disk_config.resize_increment_gb,
+            "max_disk_size_gb": self.disk_config.max_size_gb,
```

Comment:
We changed the dataclass, and not the user only needs to configure the max_size_gb. I can change the old names back. I just removed disk from the names because this is a DiskConfig the disk in the name would be redundant.